_editions/2021/tasks/sportsvideo.md

```diff
@@ -16,35 +16,33 @@ blurb: "Participants are provided with a set of videos of table tennis games and
 
 #### Evaluation methodology
-Twenty stroke classes are considered according to the rules of table tennis. This taxonomy was designed with professional table tennis teachers. We are working on videos recorded at the Faculty of Sports of the University of Bordeaux. Students are the sportsmen filmed and the teachers are supervising exercises conducted during the recording sessions. The recordings are markerless and allow the players to perform in natural conditions. 
-<!-- # Please add a sentence about how the ground truth is generated. This point is part of what makes your task attractive and interesting.-->
+Twenty stroke classes are considered according to the rules of table tennis. This taxonomy was designed with professional table tennis teachers. We are working on videos recorded at the Faculty of Sports of the University of Bordeaux. Students are the sportsmen filmed and the teachers are supervising exercises conducted during the recording sessions. The dataset has been recorded in a sport faculty facility using a light-weight equipment, such as GoPro cameras. The recordings are markerless and allow the players to perform in natural conditions from different viewpoints. These sequences were manually annotated, and the annotation sessions were supervised by professional players and teachers using a crowdsourced annotation platform.
 
-The dataset used for training is composed of videos of table tennis matches with temporal borders of performed strokes supplied in an xml file. 
-<!-- # Please edit the sentence above so it is clear which labels are released for use in training for each task. At the moment, it seems like there are not stroke labels for subtask 1-->
+The training dataset shared for each subtask is composed of videos of table tennis matches with temporal borders of performed strokes supplied in an xml file, with the corresponding stroke label.
 
-***Subtask 1: Stroke Classification:***  Participants produce an xml file where each stroke of test sequences is labeled according to the given taxonomy. Submissions will be evaluated in terms of accuracy per class and global accuracy. 
+***Subtask 1: Stroke Detection:***  The dataset used for training is composed of videos of table tennis matches with temporal borders of performed strokes supplied in an xml file. Participants are asked to temporally segment regions where a stroke is performed on unknown videos of matches. The IoU metric on temporal segments will be used for evaluation. 
```

Comment:
The sentence "The dataset used for training is composed of videos of table tennis matches with temporal borders of performed strokes supplied in an xml file." should be removed. It's a repetition of the previous one (Failed the merge between my version and the git one, if it's merge i will delete it on the next pull request, that will probably be the one also changing the Hide status to False)